src/games/rcll/challenges.clp

```diff
@@ -429,3 +432,73 @@
 		                "that was not marked as 'remaining', ignoring" crlf)
 	)
 )
+
+(defrule challenges-sync-config-with-global-field-width
+  ?cv <- (confval (path "/llsfrb/challenges/field/width") (value ?v&:(neq  ?v ?*FIELD-WIDTH*)))
```

Comment:
are these rules necessary given that the global is initialised from the config value anyways?

Reply:
yes the global can actually change when the challenge track is loaded